packages/platform/atoms/availability/wrappers/AvailabilitySettingsPlatformWrapper.tsx

```diff
@@ -49,6 +50,7 @@ export const AvailabilitySettingsPlatformWrapper = ({
   onBeforeUpdate,
   allowDelete,
   allowSetToDefault,
+  disableToasts,
```

Comment:
Would it be a good idea to set a default value of false here?

Reply:
It is optional parameter so its value is `undefined` and in the check it essentially results into false.